gulpfile.js

```diff
@@ -1,20 +1,6 @@
 var gulp = require('gulp'),
   nodemon = require('gulp-nodemon'),
-  plumber = require('gulp-plumber'),
-  livereload = require('gulp-livereload'),
```

Comment:
You should probably remove `gulp-livereload` as a dependency as well.

Reply:
I wasn't totally sure whether <https://github.com/theatlantic/arion/blob/e6da4743d760ab2200de2963c913ce681351e7f1/gulpfile.js#L6> and friends meant that we should continue to install this package.